src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1494,7 +1526,7 @@ public override BoundNode VisitCall(BoundCall node)
         {
             if (node.Method.MethodKind == MethodKind.LocalFunction)
             {
-                CheckAssigned(node.Method, node.Syntax);
+                CheckAssigned(node.Method.OriginalDefinition, node.Syntax);
```

Comment:
Why did we need to switch to `OriginalDefinition` here?  Does this affect code unrelated to local functions?